src/encoding.js

```diff
@@ -17,6 +17,9 @@ const BN = require("bn.js")
  * @return {Object[]} The decoded array of orders
  */
 function decodeOrders(bytes) {
+  if (!bytes) {
```

Comment:
nit: can this check be more explicit like `bytes === undefined || bytes === null` so things like `decodeOrders(false)` don't unexpectedly succeed?

Reply:
We can. As far as I can tell for now we only need to avoid null, but I wanted to avoid e.g. empty strings as well. But you are right that false should cause an exception. If just js had a stricter type system.